tests/Unit/RunTests.cpp

```diff
@@ -15,7 +15,8 @@
 RunTests::RunTests(CkArgMsg* msg) {
   std::set_terminate(
       []() { Parallel::abort("Called terminate. Aborting..."); });
-  printf("%s", info_from_build().c_str());
+  // clang-tidy: do not call c-style vararg functions
```

Comment:
why not call Parallel::printf

Reply:
Because we originally had this before `Parallel::printf` was a thing, and keeping it this way keeps the code more modular.